hq/app/schedule/CredentialsReportJob.scala

```diff
@@ -21,22 +21,30 @@ class CredentialsReportJob(enabled: Boolean, cacheService: CacheService)(executi
     } else {
       logger.info(s"Running scheduled job: $description")
 
-      getCredentialsReport().foreach{ case (awsAccount, result) =>
-        result.foreach{ credsReport =>
+      val someValue: List[Either[FailedAttempt, Notification]] = getCredentialsReport().toList.map{ case (awsAccount, eFCreds) =>
```

Comment:
I'm planning on testing this part of the code which has been refactored out of the foreach.